### PR TITLE
Fix markdownlint / spelling  issues #3

### PR DESCRIPTION
**Pull request by stefanroelofs**
## Proposed change

Fix several issues related to markdownlint and spelling.

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/buienradar.markdown [hunk 1]
```diff
@@ -63,23 +63,23 @@ The following entities will be created:
...
 - **Irradiance**: "Sun intensity in Watt per square meter ([W/m2](https://en.wikipedia.org/wiki/W/m2))."
 - **Rain last 24h**: The rain over the last 24 hours (in mm).
-- **Rain last hour**: The rain over the last hour (in mm). 
+- **Rain last hour**: The rain over the last hour (in mm).
 - **Temperature n days ahead**: "The forecasted temperature n days ahead (in [C](https://en.wikipedia.org/wiki/Celsius))."
 - **Minimum temperature n days ahead**: "The forecasted minimum temperature n days ahead (in [C](https://en.wikipedia.org/wiki/Celsius))."
 - **Rain chance n days ahead**: The forecasted chance for rain n days ahead (%).
 - **Sun chance n days ahead**: The forecasted chance for sun n days ahead (%).
-- **Rain n days ahead**: "The forecasted amount of rain in [mm](https://en.wikipedia.org/wiki/Millimeter) n days ahead; the average of minrain_1d and maxrain_1d."
+- **Rain n days ahead**: "The forecasted amount of rain in [mm](https://en.wikipedia.org/wiki/Millimeter) n days ahead; the average of minimum_rain_1d and maximum_rain_1d."
```
Comment (frenck) [inline source/_integrations/buienradar.markdown]: You can't just spell correct this. These are actual keys.
Do you have a PR that adjusts this in our codebase?

File: source/_integrations/buienradar.markdown [hunk 2]
```diff
@@ -63,23 +63,23 @@ The following entities will be created:
...
 - **Minimum rain n days ahead**: "The minimum forecasted amount of rain in [mm](https://en.wikipedia.org/wiki/Millimeter) n days ahead."
 - **Maximum rain n days ahead**: "The maximum forecasted amount of rain in [mm](https://en.wikipedia.org/wiki/Millimeter) n days ahead."
-- **Wind azimuth n days ahead**: Where the wind is coming from in degrees, with true north at 0° and progressing clockwise for n days ahead. (derived from winddirection_1d)
+- **Wind azimuth n days ahead**: Where the wind is coming from in degrees, with true north at 0° and progressing clockwise for n days ahead. (derived from wind_direction_1d)
```
Comment (frenck) [inline source/_integrations/buienradar.markdown]: same as above

File: source/_integrations/buienradar.markdown [hunk 3]
```diff
@@ -63,23 +63,23 @@ The following entities will be created:
...
 - **Wind direction n days ahead**: "Where the wind will be coming from n days ahead: N (North), Z (south), NO (North-East), etc."
-- **Wind force n days ahead**: "The expected windforce in [Bft](https://en.wikipedia.org/wiki/Beaufort_scale) n days ahead."
-- **Wind speed n days ahead**: "The expected windspeed in [m/s](https://en.wikipedia.org/wiki/M/s) n days ahead (derived from windforce_1d)."
+- **Wind force n days ahead**: "The expected wind force in [Bft](https://en.wikipedia.org/wiki/Beaufort_scale) n days ahead."
+- **Wind speed n days ahead**: "The expected wind speed in [m/s](https://en.wikipedia.org/wiki/M/s) n days ahead (derived from wind force_1d)."
```
Comment (frenck) [inline source/_integrations/buienradar.markdown]: same as above

File: source/_integrations/cisco_webex_teams.markdown [hunk 4]
```diff
@@ -14,17 +14,19 @@ ha_platforms:
...
-2. Log into the web client at [teams.webex.com](https://teams.webex.com/), 
- * select the room (or create a new room), 
- * then copying the room ID from the URL. 
- 
-<strong>**Note:** you must add the bot email (in the format `mybot@webex.bot`) as a participant to the room specified above.</strong>
+1. Logging in at [Cisco Webex for Developers](https://developer.webex.com/) and navigate to `Documentation`>`API Reference`>`Messages` and select List Messages, or
+2. Log into the web client at [teams.webex.com](https://teams.webex.com/),
+
+* select the room (or create a new room),
+* then copying the room ID from the URL.
```
Comment (frenck) [inline source/_integrations/cisco_webex_teams.markdown]: This is a list of item 2. This should be indented to make it appear it is part of that.

File: source/_integrations/cisco_webex_teams.markdown [hunk 5]
```diff
@@ -14,17 +14,19 @@ ha_platforms:
...
+
+**Note: you must add the bot email (in the format `mybot@webex.bot`) as a participant to the room specified above.**
```
Comment (frenck) [inline source/_integrations/cisco_webex_teams.markdown]: Let use actual notes as per developer documentation.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/buienradar.markdown (hunk 1 above)
Comment (stefanroelofs) [reply source/_integrations/buienradar.markdown]: In my configuration the entity ids are acually like this:
image
These are the default entity ids, so I assumed the documentation was wrong in this case.

File: source/_integrations/buienradar.markdown (hunk 2 above)
Comment (stefanroelofs) [reply source/_integrations/buienradar.markdown]: Actual default configuration:
image

File: source/_integrations/buienradar.markdown (hunk 3 above)
Comment (stefanroelofs) [reply source/_integrations/buienradar.markdown]: Same as above:
image

File: source/_integrations/cisco_webex_teams.markdown (hunk 4 above)
Comment (stefanroelofs) [reply source/_integrations/cisco_webex_teams.markdown]: You are very right, fixed it.

File: source/_integrations/cisco_webex_teams.markdown (hunk 5 above)
Comment (stefanroelofs) [reply source/_integrations/cisco_webex_teams.markdown]: Good idea, fixed it.

File: source/_integrations/buienradar.markdown (hunk 1 above)
Comment (frenck) [reply source/_integrations/buienradar.markdown]: Right, that is the key that is used internally nowadays I guess.

Maybe we should instead just refer to the human-readable name as is used on this page already.

The problem with entity IDs is, is that they aren't guaranteed either.

Comment (stefanroelofs) [reply source/_integrations/buienradar.markdown]: Done 👍

**Comment by stefanroelofs** [comment]
The buienradar.markdown file probably needs another review
(see commit: https://github.com/home-assistant/home-assistant.io/pull/19961/commits/17c97ca231263baa66ce7d1b0e9ccdd18470a8cc)
But otherwise this is ready to be merged.
Thanks!

File: source/_integrations/buienradar.markdown [hunk 6]
```diff
@@ -44,47 +44,49 @@ The Buienradar integration will set up separate sensor entities with more detail
...
-- **Wind azimuth n days ahead**: Where the wind is coming from in degrees, with true north at 0° and progressing clockwise for n days ahead. (derived from wind_direction_1d)
-- **Wind direction n days ahead**: "Where the wind will be coming from n days ahead: N (North), Z (south), NO (North-East), etc."
-- **Wind force n days ahead**: "The expected wind force in [Bft](https://en.wikipedia.org/wiki/Beaufort_scale) n days ahead."
-- **Wind speed n days ahead**: "The expected wind speed in [m/s](https://en.wikipedia.org/wiki/M/s) n days ahead (derived from wind force_1d)."
-- **Condition code n days ahead**: Symbol and condition code of the expected condition n days ahead.
-- **Condition n days ahead**: Symbol and expected condition n days ahead.
-- **Detailed condition n days ahead**: Symbol and detailed expected condition n days ahead.
-- **Full condition (english) n days ahead**: Symbol and full expected condition n days ahead (in English).
-- **Full condition (dutch) n days ahead**: Symbol and full expected condition n days ahead (in Dutch).
+| Entity                                | Description |
```
Comment (frenck) [inline source/_integrations/buienradar.markdown]: Don't use tables, they are really problematic on mobile devices.
This change wasn't really needed.

**Review by frenck — APPROVED** [review]
Thanks, @stefanroelofs 👍

**Pull request merged**